xml/security_ldap.xml

```diff
@@ -601,6 +601,45 @@ Instance "localhost" is running</screen>
     </para>
  </sect2>
  
+ <sect2 xml:id="sec-security-ldap-server-stop-start">
+     <title>Stopping and Starting &ds389;</title>
+     <para>
+         Use &systemd; to manage your &ds389; server instance. Get the
+         status of your server:
+     </para>
+     <screen>&prompt.user;systemctl status --no-pager --full <replaceable>dirsrv@389ds-test.service</replaceable> 
```

Comment:
You may want to template 389ds-test here with <instance name> or similar

Reply:
Sorry, not sure what you mean by template?

Reply:
Because the instance name can be changed, the command here could be `dirsrv@389ds-test` or `dirsrv@localhost` or `dirsrv@whatever_i_like`. So rather than "hardcoding" 389ds-test", in these examples I think it needs to be made clear that the admin must replace this with *their* instance name. So when I say "template" I'm suggesting something like `dirsrv@[instance_name]` so that the admin knows to replace their instance name into this location.

Reply:
Ah, ok. That is the job of the `<replaceable>` tag, which indicates user-replaceable items.